test/object-store/util/sync/baas_admin_api.cpp

```diff
@@ -849,27 +859,36 @@ AppCreateConfig default_app_config(const std::string& base_url)
         true,
     };
 
-    return AppCreateConfig{"test",
-                           base_url,
-                           "unique_user@domain.com",
-                           "password",
-                           "mongodb://localhost:26000",
-                           db_name,
-                           std::move(default_schema),
-                           std::move(partition_key),
-                           true,       // dev_mode_enabled
-                           util::none, // Default to no FLX sync config
-                           std::move(funcs),
-                           std::move(user_pass_config),
-                           std::string{"authFunc"},
-                           true,  // enable_api_key_auth
-                           true,  // enable_anonymous_auth
-                           true}; // enable_custom_token_auth
+    return AppCreateConfig{
+        "test",
+        std::move(app_url),
+        std::move(admin_url), // BAAS Admin API URL may be different
+        "unique_user@domain.com",
+        "password",
+        get_mongodb_server(),
+        db_name,
+        get_default_schema(),
+        std::move(partition_key),
+        true,                               // dev_mode_enabled
```

Comment:
do we run (pretty much) all our tests in dev mode? 🤔  I remember seeing tests where we explicitly enable it..

Reply:
Good point - I think many of our tests use the default schema or provide the schema when the app is created, and I'm not sure how many deviate from this. I can try running the tests with Dev mode disabled to see what the fallout is - if it is significant (> 5 tests), I'll disable it in a separate task.

Reply:
I was able to disable it by default without too many significant changes. There were less than 10 tests that needed to be updated.

Reply:
Nice!